cluster-autoscaler/cloudprovider/packet/packet_price_model.go

```diff
@@ -55,7 +55,7 @@ var instancePrices = map[string]float64{
 
 // NodePrice returns a price of running the given node for a given period of time.
 // All prices are in USD.
-func (model *PacketPriceModel) NodePrice(node *apiv1.Node, startTime time.Time, endTime time.Time) (float64, error) {
+func (model *EquinixMetalPriceModel) NodePrice(node *apiv1.Node, startTime time.Time, endTime time.Time) (float64, error) {
```

Comment:
Instead of renaming Packet to EquinixMetal in types, it would be cleaner if we don't stutter the package name.

i.e. `packet.PacketPriceModel` becomes `packet.PriceModel`

Reply:
https://go.dev/blog/package-names#naming-package-contents

Reply:
While we are at it `Model` is redundant to `type`.

Reply:
Done